### ISSUE TITLE
new error

### ISSUE THREAD

**Issue by zahachtah** [JobJob/D3Magic.jl#3]
After cloning D3Magic
I get error after

using D3Magic

InitError: MethodError: no method matching show(::IOContext{Base.GenericIOBuffer{Array{UInt8,1}}}, ::MIME{Symbol("text/html")}, ::String)
Closest candidates are:
  show(::IO, !Matched::MIME{Symbol("text/plain")}, ::Any) at multimedia.jl:47
  show(::IO, ::MIME{Symbol("text/html")}, !Matched::Method; kwtype) at methodshow.jl:266
  show(::IO, ::MIME{Symbol("text/html")}, !Matched::Base.MethodList) at methodshow.jl:319
  ...
during initialization of module D3Magic

First candidate most appropriate?

**Comment by JobJob** [comment]
Yeah I guess it won't work at the REPL. I  can do `using D3Magic` in Jupyter/IJulia without error, but things are pretty broken after that.

I'm sorry to say but I doubt I'll get around to fixing this in the near future. I'll update the README to reflect this.
